query-parser/src/parse.rs

```diff
@@ -163,6 +164,26 @@ def_parser!(Where, or_join, edn::ValueAndSpan, {
     })
 });
 
+def_parser!(Where, not, edn::ValueAndSpan, {
+    satisfy(|v: edn::ValueAndSpan| {
+        if let edn::SpannedValue::PlainSymbol(ref s) = v.inner {
+            s.0.as_str() == "not"
```

Comment:
So, he did and that was how things were in the [PR I did rebased against `rust`](https://github.com/mozilla/mentat/blob/f7285b268b6ea985fbe08937874ec3c3443d1758/query-parser/src/parse.rs#L139) rather than this newer branch, but it looks like he has moved to this implementation here.